packages/core/src/services/chains/ChainStreamConsumer/consumeStream.ts

```diff
@@ -59,5 +69,5 @@ export async function consumeStream({
     })
   }
 
-  return validate(chunks)
+  return parseChunks(chunks)
```

Comment:
are we changing public API? stuff that we return in the public API? if so are the changes backwards compatible?

Reply:
We're not I think. This file is responsible for sending the chunks in streaming. Which happen in the same way at the end of the file.

The return value is used internally inside `runChain`